components/courses/AssessmentsTimeline.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+  <div>
+    <v-col cols="12" md="12">
+      <div :style="chartStyle">
+        <v-chart autoresize :option="option" @click="goToAssessment" />
+      </div>
+    </v-col>
+  </div>
+</template>
+<script>
+import datetime from '@/mixins/datetime.js'
+import '@/plugins/echarts'
```

Comment:
This import is not necessary as it is automatically done by Nuxt with the new plugin for echarts that you created.